src/context.rs

```diff
@@ -903,21 +905,19 @@ impl<'a> EventCx<'a> {
             _ => (),
         }
 
-        for handler in &view.view_data().event_handlers {
```

Comment:
You should keep this handling. These are supposed to be unconditional and trigger once on any unhandled event.